src/parser.rs

```diff
@@ -72,16 +75,20 @@ named!(fname<String>, alt!(quoted | bare));
  * Header lines
  */
 
-named!(header_line_content<String>,
+named!(header_line_content<(String, Option<DateTime<FixedOffset>>)>,
     chain!(
         filename: fname ~
-        take_until!("\n") ,
+        opt!(space) ~
```

Comment:
[GVR's "spec"](http://www.artima.com/weblogs/viewpost.jsp?thread=164293) says there should be a tab character after the filename. I don't know if I've ever seen that, but we should probably accept either a space or a tab here.

Since there should only be whitespace after the filename if there is a date (or whatever else) following it, it might be better to move it into the `after` subparser instead of making it optional. But I can't think of any harm from this. What do you think?